CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+3.0.1 (2020-01-19)
```

Comment:
This should be part of `3.0.0`.  Other devs have been updating the changelog, but omitting the date to denote that it hasn't been released.

Reply:
Also, release notes for the last release you made are missing.  2.1.0 would have been another node module upgrade.